analytics/tests/test_counts.py

```diff
@@ -24,8 +24,14 @@ class AnalyticsTestCase(TestCase):
     TIME_ZERO = datetime(2042, 3, 14).replace(tzinfo=timezone.utc)
     TIME_LAST_HOUR = TIME_ZERO - HOUR
 
-    count_stat = CountStat('test stat', ZerverCountQuery(Recipient, UserCount, 'select 0'),
-                           {}, None, CountStat.HOUR, False)
+    def make_dummy_count_stat(self, current_time):
```

Comment:
Now that this is pretty specialized, let's move this to TestProcessCountStat, e.g. above assertFillStateEquals.